service/get_test.go

```diff
@@ -177,7 +180,7 @@ func Test_prepareIndexFile(t *testing.T) {
 				}
 				content := string(contentBytes)
 				count := strings.Count(content, tt.args.newRootURL)
-				if count != 3 {
+				if count != 4 {
```

Comment:
what is this magic number '4'?

Reply:
it's the number of charts in the test file. I can do better that that I'll change this.